packages/lab/stories/pill-next/pill.stories.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * **NOTE**
+ *
+ * This file has same title with the .mdx companion file.
+ * Stories defined in this/that file may not be shown in the navigation tree after hot reloading.
+ * Refresh of a page is needed.
+ */
+
+import { SaltProvider } from "@salt-ds/core";
+import { Pill as PillNext } from "@salt-ds/lab";
```

Comment:
```suggestion
import { PillNext as Pill } from "@salt-ds/lab";
```
? are you importing the correct one

Reply:
Good catch